### PR TITLE
add documentation for get_media_url (dispositionType)

### PR DESCRIPTION
**Pull request by thomasduenser**

### CODE REVIEW

File: reference/twig-extensions/functions/get_media_url.rst [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+Returns relative URL to the given media. Configuration is needed if this Twig Extension is used!
+
+**Configuration**:
+
+Following configuration means, that the default dispositionType is "attachment" for each file and only if the mimeTypes of a file match "application/pdf" or "image/jpeg" it's the "inline" dispositionType. 
+
+If the default dispositionType would be "inline" and some files should be "attachment", than the configuration of "mime_types_attachment" should be filled and "mime_types_inline" should be empty.
+
+.. code-block:: jinja
```
Comment (danrot) [inline reference/twig-extensions/functions/get_media_url.rst]: `jinja` is the templating language twig is inspired from, you should use `yaml` instead

**Comment by danrot** [comment]
Please check travis result, the build seems to be broken.

File: reference/twig-extensions/functions/get_media_url.rst [hunk 2]
```diff
@@ -0,0 +1,25 @@
+``get_media_url``
+================
+
+Returns relative URL to the given media. Configuration is needed if this Twig Extension is used!
```
Comment (wachterjohannes) [inline reference/twig-extensions/functions/get_media_url.rst]: i think it is optional but recommend (=

File: reference/twig-extensions/functions/get_media_url.rst [hunk 3]
```diff
@@ -0,0 +1,25 @@
+``get_media_url``
+================
```
Comment (wachterjohannes) [inline reference/twig-extensions/functions/get_media_url.rst]: travis said that this line is to short (-:

https://travis-ci.org/sulu-io/sulu-docs/builds/64379442#L106

**Comment by wachterjohannes** [comment]
@thomasduenser the content is ok if you answer my comments! @dantleech can you please take a look at it?

**Comment by thomasduenser** [comment]
Thanks for the feedback - updated!

**Comment by wachterjohannes** [comment]
@thomasduenser i think you have forgotten to add the file in the toc of this file https://github.com/sulu-io/sulu-docs/blob/master/reference/twig-extensions/index.rst

**Comment by thomasduenser** [comment]
@wachterjohannes Although you said it to me ... I've updated the file now!

**Pull request merged**